BookDAO updated so it wont break if no publisherID is null;

AveryGC opened the pull request:
Controllers updated to remove **, and post with designated ID

Put a try catch on the BookDAO level that reads the publisher info

Commit:
BookDAO updated so it wont break if no publisherID is null;

Controllers updated to remove **, and post with designated ID

Commit:
fixed BookService to rollback properly and also still throw exception
when error occur

Commit:
Changed the way the connection is created and passed. The connection
itself is no longer a singleton, just the connectionUtil class

srv-max commented on LMS_Admin/src/main/java/com/SS/library/Controller/AuthorController.java:
```diff
@@ -29,89 +29,110 @@
 		}
 	}
-	@RequestMapping(path = {"**/Admin/authors/{authorId}","**Admin/authors"} , method = RequestMethod.POST)
+	@RequestMapping(path = {"/Admin/authors"} , method = RequestMethod.POST)
```
URI should be lowercase

Commit:
fixed address paths,

made pathvariable the used variables in  get / delete

fixed connectionUtil so that there is no longer a single connection but
different connections being made at the service level
